recipes/volk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class VolkTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.verbose = True
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            self.run(os.path.join("bin", "test_package"))
+            self.run(os.path.join("bin", "test_package_nolibs"))
```

Comment:
Not actually needed because there is no `shared` option, or any dependency with `shared`.
But better be safe then sorry.
```suggestion
            self.run(os.path.join("bin", "test_package"), run_environment=True)
            self.run(os.path.join("bin", "test_package_nolibs"), run_environment=True)
```